src/main/java/com/artipie/management/api/ApiRepoPostRtSlice.java

```diff
@@ -0,0 +1,142 @@
+     */
+    public ApiRepoPostRtSlice(final Storage storage, final ConfigFiles configfile) {
+        this.storage = storage;
+        this.configfile = configfile;
+    }
+
+    @Override
+    public Response response(
+        final String line,
+        final Iterable<Map.Entry<String, String>> headers,
+        final Publisher<ByteBuffer> content
+    ) {
+        return new AsyncResponse(
+            new PublisherAs(content)
+                .asciiString()
+                .thenApply(form -> URLDecoder.decode(form, StandardCharsets.US_ASCII))
+                .thenApply(
+                    form -> {
+                        final Response res;
+                        final String method = ApiRepoPostRtSlice.value(form, "method");
+                        if (Method.UPDATE.value().equals(method)) {
+                            res = new ApiRepoUpdateSlice(this.configfile)
+                                .response(
+                                    line,
+                                    headers,
+                                    new Content.From(form.getBytes(StandardCharsets.US_ASCII))
+                                );
+                        } else if (Method.DELETE.value().equals(method)) {
+                            res = new ApiRepoDeleteSlice(this.storage, this.configfile)
+                                .response(
+                                    line,
+                                    headers,
+                                    new Content.From(form.getBytes(StandardCharsets.US_ASCII))
+                                );
+                        } else {
+                            res = StandardRs.NOT_FOUND;
+                        }
+                        return res;
+                    }
+                )
+        );
+    }
+
+    /**
+     * Type of method of request.
+     * @since 0.5
+     */
+    private enum Method {
+        /**
+         * Update method.
+         */
+        UPDATE("update"),
+        /**
+         * Delete method.
+         */
+        DELETE("delete");
+
+        /**
+         * String value.
+         */
+        private final String name;
+
+        /**
+         * Ctor.
+         * @param name String value
+         */
+        Method(final String name) {
+            this.name = name;
+        }
+
+        /**
+         * Method string.
+         * @return Method string.
+         */
+        public String value() {
+            return this.name;
+        }
+    }
+
+    /**
+     * Obtain value from payload, payload is a query string (not url-encoded):
+     * <code>name1=value1&name2=value2</code>.
+     * @param payload Payload to parse
+     * @param name Parameter name to obtain
+     * @return Parameter value
+     * @checkstyle StringLiteralsConcatenationCheck (10 lines)
+     */
+    private static String value(final String payload, final String name) {
+        final int start = payload.indexOf(String.format("%s=", name)) + name.length() + 1;
```

Comment:
You can use some library to parse URI queries here, and I'm suggesting to URL-encode parameters, since payload may contain `&` chars

Reply:
Do I understand correct that both parameters name and parameters may contain `&`? Or in this case it would be parsed as an argument with empty value?